### ISSUE TITLE
higher pitch audio

### ISSUE THREAD

**Issue by Cibomatto2002** [libretro/libretro-o2em#57]
I could be wrong but i don't remember it sounding high pitch in games and on the game select game screen. i'm using NTSC

**Comment by roslof** [comment]
You're not wrong. Since a change not long before your post, a change was made that affected the pitch of all audio. It is quite a bit higher than the hardware.

**Comment by Cibomatto2002** [comment]
is it a bug ? if they leave it can we see a setting to go back to what it was before?

**Comment by Cibomatto2002** [comment]
is it going to get fixed?

**Comment by roslof** [comment]
@jdgleaver the issue with global higher-pitched audio seems to have originated with this change:
https://github.com/libretro/libretro-o2em/pull/55
Merged commit: 218c9cd2

Cheers!

**Comment by roslof** [comment]
I'm hoping this project isn't dead. Hoping @jdgleaver or anybody with access to this repo can either revert the change I mentioned above or fix the issue. Since this emulator hasn't been updated in so long, I'm seeing more and more versions of this in the wild with the high-pitched audio. It's otherwise a very solid emulation of the O2/VideoPac

**Comment by LibretroAdmin** [comment]
This should be more accurate to MAME. Hopefully it is satisfactory. If not, open a new issue
